src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,9 +1,12 @@
 package mate.academy.spring.dto.request;
 
 import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 
 public class MovieRequestDto {
+    private static final int MIN_MOVIE_TITLE_LENGTH = 2;
     @NotNull
+    @Size(min = MIN_MOVIE_TITLE_LENGTH)
```

Comment:
these two can be composed with NotBlank

Reply:
If I get you right, I should change NotNull on NotBlank

Reply:
is it?